Emby.Server.Implementations/HttpServer/Security/AuthService.cs

```diff
@@ -34,7 +40,14 @@ public void Authenticate(IRequest request, IAuthenticationAttributes authAttribt
             ValidateUser(request, authAttribtues);
         }
 
-        private void ValidateUser(IRequest request, IAuthenticationAttributes authAttribtues)
+        public User Authenticate(HttpRequest request, IAuthenticationAttributes authAttributes)
+        {
+            var req = new WebSocketSharpRequest(request, null, request.Path, _logger);
```

Comment:
I was debating whether to re-implement this functionality in the api project, but I think we're going to use JWT once the migration is done, so it seemed pointless.